executor/executor.cc

```diff
@@ -572,10 +572,16 @@ void execute_one()
 	}
 
 	int call_index = 0;
+	bool usb_prog = false;
 	for (;;) {
 		uint64 call_num = read_input(&input_pos);
 		if (call_num == instr_eof)
 			break;
+		bool usb_call = false;
+		if (strcmp(syscalls[call_num].name, "syz_usb_connect") == 0) {
```

Comment:
I see that the rest of syscalls also sleep for 200ms. Which is larger than syscall timeout of 45ms. So they have no chances to finish. Do we want to extend sleep to all usb syscalls?

Reply:
Will be addressed later in another PR.